docs/upgrade-govuk-frontend.md

```diff
@@ -0,0 +1,24 @@
+# Keep this gem in sync with the Design System
+
```

Comment:
So what confused me when I started was how the Design System (& the team) links into this gem and in what way.  I wasn't sure what came first (the Design System or the gem), what should take precedence (eg the DS is _always_ the single point of truth) or where the line is between how much development can take place within this app before the DS should be involved.

Not sure if this is apt for this PR but i thought a one-liner might maker it clearer.

Eg something like:
The [Design System](https://design-system.service.gov.uk/) team are part of GDS (not GOVUK) their work heavily informs the code within this gem to create a set of UI components. This  a gem aims to document and distribute components in GOV.UK [frontend applications](https://docs.publishing.service.gov.uk/#frontend-apps)

Reply:
Fair point, Chris. The paragraph in the main README file does not contain much info around this, and people may miss it. I rephrased the first part of this doc trying to provide this context in the last commit.